src/projectFormController.js

```diff
@@ -35,7 +35,7 @@ export function createElement(type, target, id, attributes, content) {
   if (id !== 'none') {
     el.setAttribute('id', id)
   }
-  if (attributes !== undefined) {
+  if (attributes !== undefined && !attributes.hasOwnProperty('prepend')) {
```

Comment:
This could become `attributes && !attributes.hasosefosneofn()`

Reply:
gotcha, I complete spaced on the fact that you could do it this way.